applications/dds/dds_video/bootstrap.sh

```diff
@@ -1,4 +1,18 @@
 #!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights
```

Comment:
Thanks for adding missing copyright headers, looks great. For understanding, did the script catch these missing headers, and were they fixed automatically or manually?

Reply:
Yes, the script will try to detect missing ones, it can only automatically fix the minor outdated year info. Overall format still need manual/GenAI inputs